tools/bin/ecflow/processes/main/meteorological/get_metdata.py

```diff
@@ -0,0 +1,161 @@
+#!/usr/bin/env python
+#####get_metdata.py 
+#####usage: <python> <get_metdata.py> <configuration.cfg>
+
+###This script downloads meteorological data through xarray from
+###http://thredds.northwestknowledge.net:8080/thredds/reacch_climate_MET_catalog.html
+###delivered through OPeNDAP.
+
+import sys
+import numpy as np
+import xarray as xr
+import os
+import time
+import argparse
+from collections import OrderedDict
+from datetime import datetime, timedelta
+from tonic.io import read_config
+from monitor.io import replace, proc_subprocess
+
+######### ----------------------------------------###########
+
+#read in configuration file
+parser = argparse.ArgumentParser(description='Reorder dimensions')
+parser.add_argument('config_file', metavar='config_file', type=argparse.FileType('r'), nargs=1, help='configuration file')
```

Comment:
Lines should be at most 72 characters long. I suspect this is longer, and some of the attributes definitely are. For the parser command, the line can break at a comma. For the attributes (lines 51 and 53), you can use `+` to connect sub-strings.